src/LoadErrorMessage/LoadErrorMessage.tsx

```diff
@@ -107,31 +122,47 @@ const LoadErrorMessage = ({
         </Typography>
       )}
       {/* Render the contact team text */}
-      <Typography
-        variant="caption"
-        color={theme =>
-          theme.palette.mode === "dark"
-            ? lighten(theme.palette.text.secondary, 0.7)
-            : theme.palette.text.primary
-        }
-      >
-        If this persists, contact {/* Render the contact team link */}
-        {supportUrl ? (
-          <Typography
-            variant="caption"
-            component={"a"}
-            color={theme => theme.palette.primary.main}
-            sx={{ textDecoration: "underline" }}
-            onClick={handleSupportClick}
-          >
-            {contactTeam}
-          </Typography>
-        ) : (
-          <Typography variant="caption">{contactTeam}</Typography>
-        )}
-      </Typography>
+      {contactTeam !== "none" ? (
+        <Typography
+          variant="caption"
+          color={theme =>
+            theme.palette.mode === "dark"
+              ? lighten(theme.palette.text.secondary, 0.7)
+              : theme.palette.text.primary
+          }
+        >
+          If this persists, contact {/* Render the contact team link */}
+          {contactUrl ? (
+            <Link href={contactUrl} target="_blank" rel="noopener noreferrer">
+              {contactTeam}
+            </Link>
+          ) : (
+            <Typography variant="caption">{contactTeam}</Typography>
+          )}
+        </Typography>
+      ) : null}
     </Paper>
   );
 };
 
+LoadErrorMessage.default404Props = {
+  actionButtonText: "Go back",
+  contactTeam: "none",
+  image: "virto-shrugging",
+  message:
+    "We couldn't find the page you requested. It either does not exist or you may not have access to it.",
+  title: "Page not found!"
+} as const;
+
+LoadErrorMessage.defaultErrorProps = {
+  actionButtonText: "Refresh",
+  contactTeam: "Support",
+  contactUrl: "https://support.virto.com",
```

Comment:
Would have been better to default this to the actual URL we need https://www.ipg-automotive.com/en/support/support-request/. Now we need to remember to set it in every app we use where we pretty much always want this same correct URL.

Reply:
https://github.com/IPG-Automotive-UK/virto/pull/552#issuecomment-1969102717